sensor_msgs/Image image
---
yolo_object_detection/DetectedObjectArray detectedObjects